setup.py

```diff
@@ -102,6 +102,8 @@
             'weaviate-client~=3.3.0',
             'annlite>=0.3.2',
             'elasticsearch>=8.2.0',
+            'qdrant-client~=0.7.3',
+            'strawberry-graphql',
```

Comment:
why is this dependency needed?

Reply:
For now yes, If this is true, it will be solved in a different scope

Reply:
done